pydrive2/test/test_util.py

```diff
@@ -10,4 +58,19 @@ def CreateRandomFileName():
 
 
 def StripNewlines(string):
-    return newline_pattern.sub("", string)
\ No newline at end of file
+    return newline_pattern.sub("", string)
+
+
+def create_file(path, content):
+    try:
+        with open(path, "w") as f:
+            f.write(content)
+    except OSError:
```

Comment:
Is it a good idea to just ignore this? This will make the following errors cryptic.

Reply:
it seems I had an assumption that Travis runs parallel jobs on the same filesystem which can cause such errors pretty often, but, thanks, I will remove `except` to let it error out.